src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball randomBall = new Ball();
+        ColorSupplier randomColor = new ColorSupplier();
+        Random randomNum = new Random();
+
+        randomBall.number = randomNum.nextInt(101);
+        randomBall.color = randomColor.getRandomColor();

```

Comment:
Magic number '101' should be extracted into a constant. Constants are usually declared at the top of the class and have meaningful names which can help understand the purpose of the number.